src/overlay/gather-markers.ts

```diff
@@ -500,7 +501,20 @@ class OutputMarker {
     this._gatherLabel.node.textContent = 'Gather';
     (this._gatherButton.layout as PanelLayout).addWidget(this._gatherLabel);
 
+    $(this._element).css({ overflow: 'visible' });
+    
+    let element = this._element;
+    for (var i = 0; i < 3; i += 1) {
+      $(element).removeClass("p-Widget");
```

Comment:
Why does this remove a class p-Widget, and might that have unintended side effects? Instead, can you write a bit of CSS to undo CSS properties of some of the p-Widget's?

Reply:
Styling the element with `overflow: visible` has the same result as removing the `p-Widget` classes, so the action will be change the styling in the loop.

Reply:
Cool, can we do that instead? I think the ideal solution here is to set "overflow: visible" dynamically with Javascript, and make sure to set it back to the default "overflow" value as soon as the button disappears.